esti/main_test.go

```diff
@@ -142,7 +143,7 @@ func deleteAllGroups(ctx context.Context, client apigen.ClientWithResponsesInter
 			return fmt.Errorf("list groups: status: %s", resp.Status())
 		}
 		for _, group := range resp.JSON200.Results {
-			if !slices.Contains(groupsToKeep, group.Id) {
+			if !slices.Contains(groupsToKeep, swag.StringValue(group.Name)) {
```

Comment:
will it fail after the migration to group IDs?

Reply:
It shouldn't. The `groupsToKeep` is a list of group names. It can't be a list of IDs because they are known only at group creation.